File: Assets/Scirpts/Server_python/t2_TestAsyncSelectServer/proto/Messages.py
import os
import sys
sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
from net.MsgBase import MsgBase
#----------------System Message----------------
class MsgPing(MsgBase):
    def __init__(self):
        super().__init__()
        self.protoName = "MsgPing"

class MsgPong(MsgBase):
    def __init__(self):
        super().__init__()
        self.protoName = "MsgPong"
        
        
#---------------Test Message----------------
class MyMessage(MsgBase):
    def __init__(self):
        super().__init__()
        self.protoName = "MyMessage"
        self.content = ""
class MsgTest(MsgBase):
    def __init__(self):
        super().__init__()
        self.protoName = "MsgTest"
        self.str = ""
        self.num = 0

File: Assets/Scirpts/Client_cs/t2_TestAsyncSelectClient/proto/Messages.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
//---------------System Message----------------
public class MsgPing : MsgBase
{
    public MsgPing() { protoName = "MsgPing"; }
}

public class MsgPong : MsgBase 
{
    public MsgPong() { protoName = "MsgPong";  }
}


//--------------Test Message----------
public class MsgTest : MsgBase
{
    public MsgTest() { protoName = "MsgTest"; }
    public string str;
    public int num;
}